### ISSUE TITLE
KeyError executing code

### ISSUE THREAD

**Issue by milesizzo** [yhat/rodeo#38]
I pulled the latest version of the code (at time of writing) and I get the following error when I try to execute any code:

```
ERROR:rodeo.rodeo:Exception on / [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/local/lib/python2.7/dist-packages/rodeo-0.2.2-py2.7.egg/rodeo/rodeo.py", line 33, in home
    result = kernel.execute(code)
  File "/usr/local/lib/python2.7/dist-packages/rodeo-0.2.2-py2.7.egg/rodeo/kernel.py", line 139, in execute
    return self._run_code(code)
  File "/usr/local/lib/python2.7/dist-packages/rodeo-0.2.2-py2.7.egg/rodeo/kernel.py", line 134, in _run_code
    data = reply['content']['text']
KeyError: 'text'
```

**Comment by JanSchulz** [comment]
What was it you executed?

Unfortunately, sending text/plain is not always happening (the messaging spec says it should be done, but...), so maybe that line should be changed to `reply['content'].get('text', None)`?

**Comment by glamp** [comment]
I figured this would happen eventually :smile: 
:+1: for catching this (and reporting it!) 

fixed in 8834dddc9db3f5642c26f39684339972343e0a79

**Comment by milesizzo** [comment]
Sorry guys, was away all weekend. For the record, I was executing:

```print "hello"```
